actors/kinematic_character.gd

```diff
@@ -3,9 +3,12 @@ extends KinematicBody2D
 export (int) var walk_speed = 100
 export (int) var jump_height = 800
 export (int) var max_jumps = 2
+export (int) var dash_length = 300
+export (int) var dash_speed = 200
 
 var direction = 1
 var velocity = Vector2(0, 0)
+var can_dash = true
 
 onready var in_jump_speed = walk_speed
```

Comment:
move this line to https://github.com/pigdevstudio/platform_template/pull/8/files#diff-4cf3879bee4dd68b0f624d5db69cc1b9

Reply:
Should every *state related* property be moved to its corresponding *state* :thinking: